easybuild/easyconfigs/c/cdat/cdat-8.2.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,132 @@
+easyblock = 'PythonBundle'
```

Comment:
Please add patch description + link to upstream source or mention patch author (+ fix SHA256 for this patch file)